Crypto landing page copy/css/style.css

```diff
@@ -0,0 +1,995 @@
+    position: absolute;
+    z-index: -1;
+    object-fit: cover;
+}
+
+.elips h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 90px;
+    display: flex;
+    align-items: center;
+    text-transform: lowercase;
+    color: #FFFFFF;
+    position: relative;
+}
+
+.elips p {
+    margin-top: 25px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 168.5%;
+    color: #767799;
+}
+
+.elips h1::before {
+    content: '';
+    background: url('../img/strelka.png') no-repeat right top, url("../img/strelka2.png") no-repeat left bottom;
+    background-size: contain, auto;
+    width: 100px;
+    height: 53px;
+    transform: rotate(3deg);
+    position: absolute;
+    top: -42px;
+    right: 73px;
+}
+
+.section1-bottom {
+    display: flex;
+    flex-direction: row;
+    gap: 87px;
+    margin-top: 58px;
+}
+
+.section1-bottom img {
+    display: block;
+}
+
+
+.section1-bottom a {
+    display: inline-block;
+    border: 1px solid #FFFFFF;
+    text-decoration: none;
+    padding: 20px 44.5px 20px 44.5px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 21px;
+    line-height: 25px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+}
+
+/* Sectiion 2 */
+
+.section2 {
+    background: #162446;
+}
+
+.section2-wrapper {
+    display: block;
+    padding-top: 41px;
+    padding-bottom: 41px;
+
+}
+
+.section2-wrapper ul {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+/* Section 3 */
+
+.section3 {
+    display: block;
+    margin-top: 92px;
+}
+
+.section3-post h2 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 60px;
+    color: #FFFFFF;
+    text-align: center;
+}
+
+.section3-post p {
+    margin-top: 38px;
+    max-width: 924px;
+    margin-left: auto;
+    margin-right: auto;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    text-align: center;
+    color: #767799;
+}
+
+.section3-item-container {
+    margin-top: 38px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.section3-item {
+    max-width: 344px;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #C23893;
+    border-radius: 10px;
+}
+
+.section3-item img {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.section3-item h1 {
+    margin-top: 28px;
+    margin-left: 34px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 300;
+    font-size: 32px;
+    color: #FFFFFF;
+}
+
+.section3-item h2 {
+    margin-top: 38px;
+    margin-bottom: 29px;
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 33px;
+    text-align: center;
+    color: #FFFFFF;
+}
+
+.section3-item p {
```

Comment:
add separate style for the last item, currently last p tag has margin bottom 48px, it can affect design if new tag will be added after p.
add:
`.section3-item p:last-child {margin-bottom: 0;}`